docs/dev_guide/frequently-asked-questions-faq.rst

```diff
@@ -336,5 +336,14 @@ is adjusted to avoid a bit shift that is greater than the type length:
      ...
    }
 
+**WWhy some compile commands for C/C++ files are missing in the compilation database

```

Comment:
there is a FAQ:  " Why does the compilation database (compile_commands.json) not contain all source file(s) in the project? " alredy,  let's enhance the existing one.

Reply:
OKay